web/src/admin/applications/wizard/steps/providers/ApplicationWizardProviderForm.ts

```diff
@@ -16,7 +16,7 @@ import { snakeCase } from "change-case";
 import { CSSResult } from "lit";
 import { property, query } from "lit/decorators.js";
 
-export class ApplicationWizardProviderForm<T extends OneOfProvider> extends AKElement {
+export abstract class ApplicationWizardProviderForm<T extends OneOfProvider> extends AKElement {
```

Comment:
`ApplicationWizardStep` should also be abstract, but our typing on `CustomEmitterElement` makes this difficult to achieve without a lot of detangling.